src/sparseml/core/recipe/metadata.py

```diff
@@ -79,3 +80,18 @@ class RecipeMetaData(BaseModel):
     tags: List[str] = None
     target_dataset: DatasetMetaData = None
     target_model: ModelMetaData = None
+
+    def update_missing_metadata(self, other: "RecipeMetaData"):
```

Comment:
would be good to include a unit test for this function too, showing how 2 recipes with metadata get merged